src/DssSpell.t.sol

```diff
@@ -63,6 +66,82 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(dai.balanceOf(GFXLABS)        - prevDaiGfxLabs,     amountGfxLabs      * WAD);
     }
 
+    function testPayouts() public {
+        VestAbstract vest = VestAbstract(addr.addr("MCD_VEST_DAI"));
+
+        address sne = wallets.addr("SNE_WALLET");
+        address sf = wallets.addr("SF_WALLET");
+
+        uint256 snebal = dai.balanceOf(sne);
+        uint256 sfbal = dai.balanceOf(sf);
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        assertEq((dai.balanceOf(sne) - snebal) / WAD * WAD, 42917 * WAD);
+        assertEq((dai.balanceOf(sf) - sfbal) / WAD * WAD, 82417 * WAD);
```

Comment:
You might also want to check how the vow sin is affected, however that should also be affected by the delegate payments. Btw is this new thing breaking the other test?

Reply:
I think you can probably merge both tests

Reply:
yes, Tal and I are discussing in WC call